read_input_parser_cvc.l

```diff
@@ -422,12 +422,12 @@ static inline double fltlist_next_token(int * const list_end){
 ^loopextractinpath{SPC}*={SPC}*                 BEGIN(LOOPEXTRACTINPATH);
 ^loopextractfilenameprefix{SPC}*={SPC}*         BEGIN(LOOPEXTRACTFILENAMEPREFIX);
 ^loopextractfilenamesuffix{SPC}*={SPC}*         BEGIN(LOOPEXTRACTFILENAMESUFFIX);
-^loopextractntochaccumulated{SPC}*={SPC}*       BEGIN(LOOPEXTRACTNSTOCHACCUMULATED);
-^loopextractspinprojectspinproject{SPC}*={SPC}* BEGIN(LOOPEXTRACTSPINPROJECT);
+^loopextractnstochaccumulated{SPC}*={SPC}*      BEGIN(LOOPEXTRACTNSTOCHACCUMULATED);
+^loopextractspinproject{SPC}*={SPC}*            BEGIN(LOOPEXTRACTSPINPROJECT);
 ^loopextractlegacytraj{SPC}*={SPC}*             BEGIN(LOOPEXTRACTLEGACYTRAJ);
-^loopextractspinprojectspintrace{SPC}*={SPC}*   BEGIN(LOOPEXTRACTSPINTRACE);
+^loopextractspintrace{SPC}*={SPC}*              BEGIN(LOOPEXTRACTSPINTRACE);
 ^loopextractgammastructure{SPC}*={SPC}*         BEGIN(LOOPEXTRACTSPINPROJECTGAMMASTRUCTURE);
-^loopextractspinprojectgammabasis{EQL}          BEGIN(LOOPEXTRACTSPINPROJECTGAMMABASIS);
```

Comment:
```loopextractspinprojectgammabasis``` is perfect, the gamma basis should be completely irrelevant if no projection is done, please keep as is